dotnet/src/webdriver/Firefox/FirefoxDriverService.cs

```diff
@@ -103,6 +103,16 @@ protected override DriverOptions GetDefaultDriverOptions()
     /// </remarks>
     public string? LogPath { get; set; }
 
+    /// <summary>
+    /// Disable truncation of long log lines in GeckoDriver.
+    /// </summary>
+    public bool LogNoTruncate { get; set; }
```

Comment:
Kind of optional argument, would be nice to convert it to nullable `bool?`. Yes, it doesn't follow existing pattern, but improving old API is a breaking change.

Contributes to: https://github.com/SeleniumHQ/selenium/issues/14640

I am not sure, IMHO `LogTruncate` is better than `LogNoTruncate`? And Java uses it, just to aligned.

Reply:
https://github.com/SeleniumHQ/selenium/pull/16138